builtin/providers/aws/resource_aws_vpn_connection.go

```diff
@@ -57,78 +58,79 @@ func resourceAwsVpnConnection() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"vpn_gateway_id": &schema.Schema{
+			"vpn_gateway_id": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
 
-			"customer_gateway_id": &schema.Schema{
+			"customer_gateway_id": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
 
-			"type": &schema.Schema{
+			"type": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
 
-			"static_routes_only": &schema.Schema{
+			"static_routes_only": {
 				Type:     schema.TypeBool,
-				Required: true,
+				Optional: true,
```

Comment:
> Required to Optional is that if we d.Get("") on a bool and it doesn't exist 

Why wouldn't it exist if it's a `Required` attribute?

Can you elaborate why this change is needed? It seems that the below change:

```
+	} else {
+		//If there no Options on the connection then we do not support *static_routes*
+		d.Set("static_routes_only", false)
+    }

```

is what is needed here. #9110 states that on import it wasn't being set; with the above, it is. What am I missing such that we now make a once Required attribute to be optional and computed?